Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -46,17 +46,18 @@ extension Snapshotting where Value == URLRequest, Format == String {
     var components = ["curl"]
 
     // HTTP Method
-    let httpMethod = request.httpMethod!
-    switch httpMethod {
-    case "GET": break
-    case "HEAD": components.append("--head")
-    default: components.append("--request \(httpMethod)")
+    if let httpMethod = request.httpMethod {
+      switch httpMethod {
+      case "GET": break
+      case "HEAD": components.append("--head")
+      default: components.append("--request \(httpMethod)")
+      }
     }
 
     // Headers
     if let headers = request.allHTTPHeaderFields {
       for field in headers.keys.sorted() where field != "Cookie" {
-        let escapedValue = headers[field]!.replacingOccurrences(of: "\"", with: "\\\"")
+        guard let escapedValue = headers[field]?.replacingOccurrences(of: "\"", with: "\\\"") else { continue }
```

Comment:
Let's also leave the force unwrap in here, after all `field` can only be a key in the `headers` due to us looping over it:

```suggestion
        let escapedValue = headers[field]!.replacingOccurrences(of: "\"", with: "\\\"")
```